### PR TITLE
Fix docs issues and create a new error page

### PR DESCRIPTION
**Pull request by rkuchan**
Small changes:

* Fixes the typo in `alert_node`, #782 
* Adds `continuous query name` and `subscription name` to the list of identifiers in the glossary

Larger changes:

* Creates a new `Error Messages` page and moves error-related FAQs to it
* Adds a description of the `bad timestamp` error to the `Error Messages` page, #793
* Adds a description of the `cache maximum memory size exceeded` error to the `Error Messages` page, #773

### CODE REVIEW

File: content/kapacitor/v0.13/nodes/alert_node.md [hunk 1]
```diff
@@ -1096,7 +1096,7 @@ Example:
            .slack()
 ```
```
Comment (beckettsean) [inline content/kapacitor/v0.13/nodes/alert_node.md]: there's the fix!

the whitespace changes are brutal!

File: content/influxdb/v1.0/troubleshooting/errors.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  influxdb_1_0:
+    weight: 30
+    parent: troubleshooting
+---
+
+This page documents parsing errors, their descriptions, and, where applicable,
+common resolutions.
+
+<dt>
+**Disclaimer:** This document is under development and does not contain a
```
Comment (beckettsean) [inline content/influxdb/v1.0/troubleshooting/errors.md]: **Disclaimer:** This document does not contain an exhaustive list of all possible InfluxDB errors.

File: content/influxdb/v1.0/troubleshooting/errors.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+comprehensive list of all InfluxDB errors.
+</dt>
+
+## error parsing query: found < >, expected identifier at line < >, char < >
+
+### InfluxQL Syntax
+The `expected identifier` error occurs when InfluxDB anticipates an identifier
+in a query but doesn't find it.
+Identifiers are tokens that refer to database names, retention policy names,
+continuous query names, user names, measurement names, tag keys, and field keys.
```
Comment (beckettsean) [inline content/influxdb/v1.0/troubleshooting/errors.md]: subscriptions?

File: content/influxdb/v1.0/troubleshooting/errors.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+> SELECT * FROM WHERE "blue" = true
+ERR: error parsing query: found WHERE, expected identifier at line 1, char 15
+```
+Query 2 is missing a measurement name between `FROM` and `WHERE`.
+
+### InfluxQL Keywords
+In some cases the `expected identifier` error occurs when one of the
+[identifiers](/influxdb/v1.0/concepts/glossary/#identifier) in the query is an
+[InfluxQL Keyword](/influxdb/v1.0/query_language/spec/#keywords).
+To successfully query an identifier that's also a keyword enclose that
```
Comment (beckettsean) [inline content/influxdb/v1.0/troubleshooting/errors.md]: To successfully query an identifier that's also a keyword, enclose that

File: content/influxdb/v1.0/concepts/glossary.md [hunk 5]
```diff
@@ -77,7 +77,7 @@ See [InfluxQL Functions](/influxdb/v1.0/query_language/functions/) for a complet
 Related entries: [aggregation](/influxdb/v1.0/concepts/glossary/#aggregation), [selector](/influxdb/v1.0/concepts/glossary/#selector), [transformation](/influxdb/v1.0/concepts/glossary/#transformation)  
 
 ## identifier
-Tokens which refer to database names, retention policy names, user names, measurement names, tag keys, and field keys.
+Tokens which refer to database names, retention policy names, continuous query names, user names, measurement names, tag keys, and field keys.
```
Comment (beckettsean) [inline content/influxdb/v1.0/concepts/glossary.md]: subscription names?

**Comment by beckettsean** [comment]
Looks good. A few minor notes but nothing critical.

**Comment by rkuchan** [comment]
@beckettsean - one more error message entry up for review, otherwise this should be good to go!

**Comment by beckettsean** [comment]
looks great!

**Pull request merged**